eventually/eventually/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-
+Temp_Path = os.path.realpath('.')
```

Comment:
Why does this variable contain camelcase and snake-case convention? `temp_path` or `TempPath`

Reply:
This variable is not used. I deleted it